packages/experiments/src/components/Stack/Stack.tsx

```diff
@@ -49,7 +49,11 @@ const view = (props: IViewComponentProps<IStackProps, IStackStyles>) => {
     }
   );
 
-  return <RootType className={classNames.root}>{stackChildren}</RootType>;
+  return (
+    <RootType {...rest} className={classNames.root}>
```

Comment:
What is the purpose of adding `{...rest}`? Just curious, as that seems to be adding attributes like `horizontalAlignment`, `verticalAlignment`, and `grow` to the DOM structure.

Reply:
data and aria attributes, in addition to any other props that are passed in. E.g. 

```
<Stack as='button' onClick={ click }>
```